app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,15 @@
+class GroupsController < ApplicationController
+  def assign
+    user = current_user
+    user.group_id = find_group(params[:address])
+    user.save
```

Comment:
Do we want to write a case for when (for whatever reason) the group doesn't save?